dev-tools/devTools.py

```diff
@@ -183,7 +183,7 @@ def _maybeCreateDiagrams(arguments):
         return
 
     diagramsFormat = arguments.diagramsFormat if arguments.diagramsFormat else "pdf"
-    cmd = f"{_SCRIPTS_DIR / 'pyreverse'} -k -o {diagramsFormat} -p optihood -d test-results optihood"
+    cmd = f"{_SCRIPTS_DIR / 'pyreverse'} -o {diagramsFormat} -p optihood -d test-results optihood"
     _printAndRun(cmd.split())
```

Comment:
Pyreverse ignores class attributes and methods when "-k" is used.
The new implementation thus provides better feedback.